README.md

```diff
@@ -46,31 +48,81 @@ This document is for the latest Aimeos TYPO3 **21.10 release and later**.
 
 The latest version can be installed via composer too. This is especially useful if you want to create new TYPO3 installations automatically or play with the latest code. You need to install the composer package first if it isn't already available:
 
-`php -r "readfile('https://getcomposer.org/installer');" | php -- --filename=composer`
+```bash
+php -r "readfile('https://getcomposer.org/installer');" | php -- --filename=composer
+```
 
 In order to tell install TYPO3, you have to execute
 
-`composer create-project typo3/cms-base-distribution myshop`
+```bash
+composer create-project typo3/cms-base-distribution myshop
+
+# If you want to use a specific TYPO3 version, use this:
+composer create-project "typo3/cms-base-distribution:^11" myshop
+```
 
 This will install TYPO3 into the `./myshop/` directory.
 
-Then, change to the `./mshop/` directory and install the Aimeos extension for TYPO3 with:
+Change to the `./mshop/` directory and add these lines to your composer.json, if they are not already there:
 
-`composer req aimeos/aimeos-typo3:~22.10`
+```json
+"minimum-stability": "dev",
+"prefer-stable": true,
+```
```

Comment:
With the v15.0 release of the `graphql-php`, this is optional now so it would be better to add something like:

"If composer is unable to install one or more packages because the required minimum stability isn't met, add this to your composer.json"

Reply:
You know better, so whatever text is convenient for you, as long as it is mentioned.